tests/util/color_test.py

```diff
@@ -0,0 +1,44 @@
+from sys import stdout
+
+from detect_secrets.util.color import AnsiColor
+from detect_secrets.util.color import colorize
+
+
+def colorize_enabled(text: str, color: AnsiColor) -> str:
+    return '\x1b{}{}\x1b{}'.format(
+        color.value,
+        text,
+        AnsiColor.RESET.value,
+    )
+
+
+def expect_enabled(text: str):
+    for color in AnsiColor:
+        expected = colorize_enabled(text, color)
+        assert colorize(text, color) == expected
+
+
+def expect_disabled(text: str):
+    for color in AnsiColor:
+        assert colorize(text, color) == text
+
+
+def test_colorize_1(monkeypatch):
```

Comment:
Can we update to a more descriptive name? Possibly: `test_colorize_enabled_terminal_disabled_piped`